tests/test_commands.py

```diff
@@ -135,3 +135,16 @@ def test_who(fake_message, mocker):
     mock_reply_to.assert_called_once_with(
         fake_message, expected_text, disable_preview=True
     )
+
+
+def test_hotline(fake_message, mocker):
+    fake_message.text = '/hotline'
+    mock_hotline_message_id = mocker.patch('tululbot.commands.HOTLINE_MESSAGE_ID',
+                                           autospec=True)
```

Comment:
I think it's not necessary to set `autospec=True` here. Could you please remove this and see if it's still ok?